storage/client/client.go

```diff
@@ -1568,6 +1592,17 @@ func (c *StorageClient) RuntimeAccount(ctx context.Context, address staking.Addr
 		return nil, wrapError(err)
 	}
 
+	// Reconcile sdk balance from node with the balance fetched from the db.
+	nodeBalance := <-ch
+	if nodeBalance != nil {
+		if len(a.Balances) != 1 {
+			c.logger.Warn("multiple sdk balance denominations detected; skipping node balance check", "address", address)
```

Comment:
How come

you can get all balances from sdk (https://github.com/oasisprotocol/explorer/pull/1318#discussion_r1531434363)

Reply:
Oh thanks for the pointer; didn't realize that. Updating now